### PR TITLE
ISSUE-21 LSC job to synchronize quota size for users

### PR DESCRIPTION
**Pull request by quantranhong1999**

### CODE REVIEW

File: src/test/resources/lsc-conf/logback.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
 			<layout class="org.lsc.utils.output.LdifLayout">
 				<Pattern>%date{MMM dd HH:mm:ss} - %-5level - [%file : %line] - %message%n</Pattern>
 			</layout>
-		</encoder>
+		</encoder>w
```
Comment (chibenwa) [inline src/test/resources/lsc-conf/logback.xml:11]: Oups?

**Review by chibenwa — COMMENTED** [review]
Looks super cool.

Awesome work!

**Review by Arsnael — COMMENTED** [review]
Looks good, nothing to add

**Comment by Arsnael** [comment]
It's an other lsc test here failing =>

[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   JamesForwardDstServiceTest.beforeAll:146->createUser:656 1 expectation failed.
Expected status code <204> but was <431>.

But it's weird, should check it.

**Comment by quantranhong1999** [comment]
I have green build (multiple times) with java 11 build:
image

> [ERROR] JamesForwardDstServiceTest.beforeAll:146->createUser:656 1 expectation failed.
Expected status code <204> but was <431>.

Running the build with java 21 would help me easily get this too. So my guess this is a CI worker issue. I do not change any code regarding the forward sync anyway.

How about just merge this and create another task to resolve the build issue?

**Comment by quantranhong1999** [comment]
> So my guess this is a CI worker issue

Or not. I check out the main branch and run tests repeat until failure, can produce the issue within `JamesForwardDstServiceTest`. So this errors happens when many tests executed, likely a test isolation issue.

File: pom.xml [hunk 2]
```diff
@@ -56,7 +56,10 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-surefire-plugin</artifactId>
-				<version>2.22.0</version>
+				<version>3.5.2</version>
+				<configuration>
+					<reuseForks>false</reuseForks>
```
Comment (quantranhong1999) [inline pom.xml:61]: Green build.

**Pull request merged**